### PR TITLE
[RFC] Hide root taxon from UI

### PR DESCRIPTION
**Pull request by michalmarcinkowski**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets |  |
| License         | MIT |

Do not allow to navigate to a root taxon in the default UI.

### CODE REVIEW

**Comment by okwinza** [comment]
How about showing it but without wrapping it in <a> tag?

**Comment by pjedrzejewski** [comment]
@okwinza +1.

**Comment by michalmarcinkowski** [comment]
Done

File: src/Sylius/Bundle/ShopBundle/Resources/views/Taxon/Show/_breadcrumb.html.twig [hunk 1]
```diff
@@ -4,8 +4,12 @@
     <a href="{{ path('sylius_shop_homepage') }}" class="section">{{ 'sylius.ui.home'|trans }}</a>
     <div class="divider"> / </div>
     {% for parent in parents %}
-    <a href="{{ path('sylius_shop_taxon_show', {'slug': parent.slug}) }}" class="section">{{ parent.name }}</a>
-    <div class="divider"> / </div>
+        {% if  parent.isRoot %}
```
Comment (okwinza) [inline src/Sylius/Bundle/ShopBundle/Resources/views/Taxon/Show/_breadcrumb.html.twig]: double space

**Comment by pjedrzejewski** [comment]
👍 👍 👍

**Pull request merged**